src/cmd-koji-upload

```diff
@@ -429,12 +429,10 @@ class Reserve(_KojiBase):
         """
         log.info("Reserving a unique koji id")
 
```

Comment:
since we are now doing something interesting here when it comes to version and release let's add a comment and the top (I'm sorry I know I asked you to remove the `release` variable in an earlier iteration of this PR:

```
# The koji/brew NVR is constructed like so:
# Name = "rhcos-$arch", like `rhcos-x86_64`
# Version = Everything before `-` in RHCOS version
# Release = Everything after `-` in RHCOS version
# 
# Example: RHCOS Build ID: 414.92.202307170903-0 for x86_64
#   Name = rhcos-x86_64
#   Version = 414.92.202307170903
#   Release = 0
#   NVR = rhcos-x86_64-414.92.202307170903-0
version, release = build.build_id.split('-')
```